.travis.yml

```diff
@@ -31,15 +31,12 @@ jobs:
   fast_finish: true
 
   include:
-    # Minimal tests set: SQLite on PHP 5.6/7.0, MySQL 5.7 on PHP 7.0 and code sniffer.
-    - stage: minimal
-      php: 5.6
-      env: "DB=sqlite db_dsn='sqlite:///tmp/test.sql'"
+    # Minimal tests set: SQLite on PHP 7.2, MySQL 5.7 on PHP 7.1 and code sniffer.
     - stage: minimal
       php: 7.2
       env: "DB=sqlite db_dsn='sqlite:///tmp/test.sql'"
     - stage: minimal
-      php: 7.0
+      php: 7.1
```

Comment:
Out of the scope: but why not add a `postgres` stage with 7.1 or 7.2  in `minimal` to get related errors sooner?